src/rust/src/x509/verify.rs

```diff
@@ -118,6 +124,34 @@ impl PolicyBuilder {
         })
     }
 
+    fn build_client_verifier(&self, py: pyo3::Python<'_>) -> CryptographyResult<PyClientVerifier> {
+        let store = match self.store.as_ref() {
+            Some(s) => s.clone_ref(py),
+            None => {
+                return Err(CryptographyError::from(
+                    pyo3::exceptions::PyValueError::new_err(
+                        "A client verifier must have a trust store.",
+                    ),
+                ));
+            }
+        };
+
+        let time = match self.time.as_ref() {
+            Some(t) => t.clone(),
+            None => datetime_now(py)?,
+        };
+
+        let policy = OwnedPolicy::try_new(SubjectOwner::None, |_| {
+            Ok::<PyCryptoPolicy<'_>, pyo3::PyErr>(PyCryptoPolicy(Policy::client(
```

Comment:
Can't `Policy::client` have a `'static` lifetime, and so this dance isn't required?

Reply:
Yes, thanks!